h2d/Scene.hx

```diff
@@ -511,7 +511,7 @@ class Scene extends Layers implements h3d.IDrawable implements hxd.SceneEvents.I
 			target = new Tile(tex,0, 0, width, height);
 		}
 		engine.begin();
-		engine.setRenderZone(target.x, target.y, target.width, target.height);
+		engine.setRenderZone(Std.int(target.x), Std.int(target.y), Std.int(target.width), Std.int(target.height));

```

Comment:
I guess you should use  x0 = Math.floor(target.x)  , iwidth = Math.ceil(target.x + target.width) - x0

Reply:
I think about adding `public var ix/iy/iwidth/iheight(get, never):Int` that would return `Std.int/Math.ceil` as a helper properties to Tile. Would save a lot of Float->Int conversion clutter.

Reply:
I don't think that's best. depending on the condition you might want to floor/ceil